library/core/tests/intrinsics.rs

```diff
@@ -99,3 +99,22 @@ fn test_const_deallocate_at_runtime() {
         const_deallocate(core::ptr::null_mut(), 1, 1); // nop
     }
 }
+
+#[cfg(not(bootstrap))]
+#[test]
+fn test_three_way_compare_in_const_contexts() {
```

Comment:
Can you add `const` tests as well, to ensure the const_eval code path is covered?

Reply:
Eh, I didn't see the `const` block, sorry for that. I shouldn't do reviews when I am so tired, it seems.